github/Installation.pyi

```diff
@@ -8,5 +8,5 @@ class Installation(NonCompletableGithubObject):
     def _initAttributes(self) -> None: ...
     def _useAttributes(self, attributes: Dict[str, Any]) -> None: ...
     @property
-    def id(self) -> _ValuedAttribute: ...
```

Comment:
You can drop the import on line 3 too, please.

Reply:
Ta, missed that